ngi_reports/ngi_reports.py

```diff
@@ -52,105 +105,235 @@ def make_reports (report_type, working_dir=os.getcwd(), config_file=None, **kwar
+        help="Configuration file to use instead of default (~/.ngi_config/ngi_reports.conf)",
+    )
+    parser.add_argument(
+        "-p",
+        "--project",
+        default=None,
+        action="store",
+        help="Project name to generate 'project_summary' report",
+    )
+    parser.add_argument(
+        "-s",
+        "--signature",
+        default=None,
+        action="store",
+        help="Signature/Name for person who generates 'project_summary' report",
+    )
+    parser.add_argument(
+        "-q",
+        "--quality",
+        default=None,
+        action="store",
+        type=int,
+        help="Q30 threshold for samples to set status",
+    )
+    parser.add_argument(
+        "-y",
+        "--yield_from_fc",
+        default=False,
+        action="store_true",
+        help="Compute the total for each sample from the retrived FC directly",
+    )
+    parser.add_argument(
+        "--skip_fastq",
+        action="store_true",
+        help="Option to skip naming convention of fastq files from report",
+    )
+    parser.add_argument(
+        "--exclude_fc",
+        nargs="*",
+        default=[],
+        action="store",
+        help="Exclude these FCs while processing, Format should be BH3JLWCCXX/000000000-AEUUP.",
+    )
+    parser.add_argument(
+        "--no_txt",
+        action="store_true",
+        help="Use this option to not generate TXT files for tables",
+    )
+    parser.add_argument(
+        "--samples",
+        default=None,
+        action="store",
+        nargs="*",
+        help="Limit the samples to include in reports",
+    )
+    parser.add_argument(
+        "--samples_extra",
+        default={},
+        action="store",
+        type=json.loads,
+        help='Pass in extra information about samples as a json string, having each sample as a key. Example: --samples_extra \'{"TS001-1": {"delivered": "20150701"}}\'',
+    )
+    parser.add_argument(
+        "--fc_phix",
+        default={},
+        action="store",
+        type=json.loads,
+        help='Overwrite or use Phix values for mentioned flowcells/lanes provided as a json string, having each flowcell as a key. Example: --fc_phix \'{"BH3JLWCCXX": {"1": "0.42", "3": "0.46"}}\'',
+    )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version="NGI reports version - {}".format(__version__),
+    )
+    parser.add_argument(
+        "-md",
+        "--markdown_file",
+        default=None,
+        help="Regenerate the html report from the given markdown file",
+    )
+    parser.add_argument(
+        "-b",
+        "--barcode_from_fc",
+        default=False,
+        action="store_true",
+        help="retrieve the index from the FC directly",
+    )
 
     kwargs = vars(parser.parse_args())
 
-    if kwargs['markdown_file']:
-        print('HTML report written to: '+markdown_to_html(kwargs['report_type'], markdown_path=kwargs['markdown_file']))
+    if kwargs["markdown_file"]:
```

Comment:
When this is true, the working directory check is not used. Is that intentional?

Reply:
It's not per-se intentional, but it is a different use case, for which we, or at least I, am anyway in the correct path.

Reply:
Looking at it closer, it doesn't matter in which directory you are when you make the html from markdown file, since it is always placed in the same directory as the .md file, see line 203: